public/locales/en/gear.json

```diff
@@ -0,0 +1,3 @@
+{
+    "RefreshAvailability": "Roll for availability"
+}
```

Comment:
Needs a blankline at the end.

https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file